src/SearchInput.tsx

```diff
@@ -0,0 +1,31 @@
+import { css } from "emotion";
+import React, { ChangeEvent } from "react";
+
+const inputCss = css`
+  justify-self: flex-end;
+  width: 200px;
+  height: 29px;
+  padding: 0 5px;
+  border: 3px solid #71997f;
```

Comment:
We shouldn't be hardcoding the colors. I have a color map that I've been using. It can be ejected into a seperate file if needed.